Source/WebCore/css/CSSCustomPropertyValue.h

```diff
@@ -88,7 +88,7 @@ class CSSCustomPropertyValue final : public CSSValue {
         return adoptRef(*new CSSCustomPropertyValue(name, VariantValue { WTFMove(syntaxValueList) }));
     }
 
-    String customCSSText() const;
+    WEBCORE_EXPORT String customCSSText() const;
```

Comment:
This is wrong. For the test we should call `cssText`, which will call this.

Reply:
Fixed. Wasn't previously aware of `cssText()` which then called `customCSSText()` for the appropriate child class.